src/daemons/MetaDaemon.cpp

```diff
@@ -45,6 +45,68 @@ static std::unique_ptr<apache::thrift::ThriftServer> gServer;
 static void signalHandler(int sig);
 static Status setupSignalHandler();
 
+std::unique_ptr<nebula::kvstore::KVStore> initKV(std::vector<nebula::HostAddr> peers,
+                                                 nebula::HostAddr localhost) {
+    auto partMan
```

Comment:
std::vector<nebula::HostAddr> &peers and nebula::HostAddr &localhost

Reply:
Good point.